internal/mongo/voting.go

```diff
@@ -0,0 +1,57 @@
+package mongo
+
+import (
+	"context"
+	"time"
+
+	"github.com/Coflnet/sky-mayor/internal/model"
+	"github.com/rs/zerolog/log"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func InsertVoting(voting *model.Voting) error {
+	ctx, _ := context.WithTimeout(context.Background(), time.Second*30)
+
+	voting.ID = primitive.NewObjectID()
+
+	result, err := votingCollection.InsertOne(ctx, voting)
+	if err != nil {
+		log.Error().Err(err).Msgf("there was an error when updating voting with id %s", voting.ID)
+		return err
+	}
+
+	log.Info().Msgf("%v id, voting (timestamp %s) was inserted", result.InsertedID, voting.Timestamp.Format("2006-01-02 15:04:05"))
+	return nil
+}
+
+func GetLastVoting() (*model.Voting, error) {
```

Comment:
use a limit 1 or findOne method, otherwise you will load a complete batch of documents